spec/features/visitor_can_view_items_spec.rb

```diff
@@ -1,25 +1,28 @@
 require 'rails_helper'
+# include ActionView::Helpers::TextHelper
 
 RSpec.feature "visitor can view items" do
   scenario "index displays all current items for visitor" do
 
-    db_repo = FactoryJordan.new
-    db_repo.create_travesty(1)
-    travesty = db_repo.travesties
-    item_1 = db_repo.items[0]
-    item_2 = db_repo.items[1]
+    travesties = create_list(:travesty_with_items, 2)
+
+    items = (item_1, item_2, item_3, item_4 = travesties[0].items)
```

Comment:
Useless assignment to variable - `item_1`. Use `_` or `_item_1` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_2`. Use `_` or `_item_2` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_3`. Use `_` or `_item_3` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `item_4`. Use `_` or `_item_4` as a variable name to indicate that it won't be used.